backends/dummy/lib.rs

```diff
@@ -36,24 +37,24 @@ impl BackendInit for DummyBackend {
 }
 
 impl Backend for DummyBackend {
-    fn create_audiostream(&self) -> Box<MediaStream> {
-        Box::new(DummyMediaStream)
+    fn create_audiostream(&self) -> MediaStreamId {
+        MediaStreamId::new()
```

Comment:
Should the dummy backend be registering dummy streams instead? `register_stream(Box::new(DummyStream))`?